sources/Inputs/src/Configuration.cpp

```diff
@@ -81,6 +81,24 @@ updateActivePowerCompensationValue(Configuration::ActivePowerCompensation& activ
   }
 }
 
+/**
+ * @brief Helper function to update the chosen outputs
+ *
+ * @param chosenOutputs the chosen outputs to update
+ * @param tree the element of the boost tree
+ */
+void
+updateChosenOutput(std::vector<std::string>& chosenOutputs, const boost::property_tree::ptree& tree) {
+    std::vector<std::string> chosenOutputsList;
+    const boost::optional<const boost::property_tree::ptree &>& optionalChosenOutputs = tree.get_child_optional("chosenOutputs");
+    if (optionalChosenOutputs) {
+      for (auto& chosenOutput : *optionalChosenOutputs) {
+        chosenOutputsList.push_back(chosenOutput.second.get_value<std::string>());
```

Comment:
why not directly pushing back in the chosenOutputs parameter?

Reply:
Ce code n'est plus d'actualité